### ISSUE TITLE
Feature Request: wipe existing tasks

### ISSUE THREAD

**Issue by leira-sanchez** [huytd/pomoday-v2#56]
I'd like a command to wipe all the existing tasks so I can start fresh. Also, a way to reset the numbers would be cool.

**Comment by huytd** [comment]
Yes that can be added.

For reseting the numbers, it's a bit tricky because the number need to be picked up from the latest one, regardless to the status of the tasks.

For now, If you get clustered between the unfinished and finished tasks, you can try the `hide` command, for example `hide done`.

**Comment by leira-sanchez** [comment]
Yeah, the numbers part is tricky. My thought is that if I keep using this app so much, I'm going to get into big numbers (1000+) and it'll be harder to do c 21243, d 13452, b 34235, etc.

Love the app! Great job, guys!
